CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 - Fixed a bug in the discretisation of initial conditions of a scaled variable ([#2856](https://github.com/pybamm-team/PyBaMM/pull/2856))
 - Fixed keyerror on "all" when getting sensitivities from IDAKLU solver([#2883](https://github.com/pybamm-team/PyBaMM/pull/2883))
+- Initial conditions for sensitivity equations calculated correctly ([#2920](https://github.com/pybamm-team/PyBaMM/pull/2920))
```

Comment:
needs to be moved to new section

Reply:
its a bug fix isn't it? Previous result was wrong, this PR fixes it.

Reply:
I thought it was in the wrong month, my bad